_plugins/parseindico.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'net/http'
+require 'uri'
+require 'json'
+require 'yaml'
+require 'date'
+
+
+module Indico
+
+  # Look for topical meetings
+  class Meetings
+    attr_accessor :dict
+
+    # ID for IRIS-HEP: 10570
+    def initialize(indico_id)
+      uri = URI.parse("https://indico.cern.ch/export/categ/#{indico_id}.json?pretty=no") # from=today&
+      response = Net::HTTP.get_response(uri)
+
+      string = response.body
+      parsed = JSON.parse(string) # returns a hash
+
+      @dict = {}
+
+      parsed['results'].each do |i|
```

Comment:
hi @henryiii - at least when writing files, I needed
i["title"]=i["title"].gsub(':','-')
to avoid : in the yml fields.

and to avoid worries of mutliline descriptions
res["description"]=''#i["description"]

Reply:
That shouldn't be a problem unless we actually write yaml files; the correct thing should be in memory. I can check it for yaml file writing, though...

Reply:
```yaml
name: 'FuncX: High Performance Function as a Service for Science'
```

That's valid YAML, correct?

Reply:
Do we have a multiline description I can check?